plugins/device/api/resp.go

```diff
@@ -2,17 +2,18 @@ package api
 
 import (
 	"github.com/gluster/glusterd2/pkg/api"
+
+	"github.com/pborman/uuid"
 )
 
 // Info represents structure in which devices are to be store in Peer Metadata
 type Info struct {
-	Name          string `json:"name"`
-	State         string `json:"state"`
-	VgName        string `json:"vg-name"`
-	AvailableSize uint64 `json:"available-size"`
-	ExtentSize    uint64 `json:"extent-size"`
-	Used          bool   `json:"used"`
-	PeerID        string `json:"peer-id"`
+	Device        string    `json:"device"`
```

Comment:
@aravindavk I would like to know why Vgname was removed, it would be useful to know the VG of device via the device struct itself.